python/tvm/testing.py

```diff
@@ -379,7 +379,7 @@ def _get_targets(target_str=None):
     if len(target_str) == 0:
         target_str = DEFAULT_TEST_TARGETS
 
-    target_names = set(t.strip() for t in target_str.split(";") if t.strip())
+    target_names = list(sorted(set(t.strip() for t in target_str.split(";") if t.strip())))
```

Comment:
A thought from this morning.  Do we want to have the list in sorted order, or in the order specified by `target_str`?  Either way would be deterministic, but I'd lean toward the latter so that we can have explicit control over the order of targets.  We can get this behavior if we use a `dict` instead of a `set` for removing duplicates.

`target_names = list({t.strip():None for t in target_str.split(';') if t.strip()})`

Reply:
ah yeah probably in order of target_str